content/en/Platform Deep Dive/Organization/automated-data-removal.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Automated Data Removal"
+linkTitle: "Automated Data Removal"
+weight: 40
+description: >
+  Process of automatically deleting data from the Cobalt Platform based on predefined rules and schedules.
+---
+
+{{% pageinfo %}}
+When your organization's contract expires, the associated data will be removed after a predefined period of time. This period can be extended based on certain conditions. Learn more about our [privacy policy](https://cobalt.io/terms#privacy).
+{{% /pageinfo %}}
+
+When your contract ends, a 15-day grace period begins during which you will still have access to the platform to [export any necessary data](/platform-deep-dive/organization/your-contract/#download-your-organizations-data). Organization Owners will also receive a notification about this. If the contract is renewed, full access to your account will be restored.
```

Comment:
It just came to me - I wasn't sure if we did send an email notification for contract end or the grace period. In this [document](https://zombie.atlassian.net/wiki/spaces/FND/pages/2616164635/Revoke+App+Access+to+Churned+Orgs+Product+Requirements), it says 'No email notifications to start - only in platform notifications.' and i can't find any email templates in Figma or Mandrill either.

Reply:
Good catch, I forgot it's only 60 days before data removal that the first notification goes out. Will remove this sentence.